Api call on login

scauror opened the pull request:
Added:
	- Login Button on HeaderBar now responsive if logged in
	- Login View api call when clicking login button
	- Error message if credentials don't match existing user in DB

Modified:
	- UnknownPage enhanced with a sentence

scauror committed:
Api call on login

Added:
	- Login Button on HeaderBar now
		responsive if logged in
	- Login View api call when clicking
		login button
	- Error message if credentials don't
		match existing user in DB

Modified:
	- UnknownPage enhanced with a sentence

Dyamen1411 reviewed (changes requested):
The login payload should have the actual username and password from the fields, not hardcoded values...
There are indentation errors.

scauror committed:
Merge branch 'dev' into api_call_refacto

scauror committed:
HeaderBar Responsive / LoginView functional

Added:
	- Login Button on HeaderBar now responsive if logged in
	- Login View api call when clicking login button
	- Error message if credentials don't match existing user in DB

scauror commented:
Payload Usage fixed
Logout using 'deauthentificate'

Dyamen1411 reviewed (approved):
Everything looks fine.
TODO: uppon cach refresh, the header bar will show that you are logged even when you are now. This is a purely visual bug but it should be addressed ASAP.
